### PR TITLE
fix: minor issues

### PR DESCRIPTION
**Pull request by wep21**

### CODE REVIEW

File: mcap_vendor/CMakeLists.txt [hunk 1]
```diff
@@ -68,13 +68,6 @@ macro(build_mcap_vendor)
   )
 
   install(TARGETS mcap EXPORT export_mcap)
-
-  install(
-    EXPORT export_mcap
-    FILE mcap-config.cmake
-    DESTINATION share/cmake/mcap/
-    NAMESPACE "mcap_vendor::"
-    EXPORT_LINK_INTERFACE_LIBRARIES)
```
Comment (jtbandes) [inline mcap_vendor/CMakeLists.txt:77]: Why remove this? Do we need to keep `install(TARGETS mcap EXPORT export_mcap)`?

File: rosbag2_storage_mcap/CMakeLists.txt [hunk 2]
```diff
@@ -35,21 +34,21 @@ target_compile_features(${PROJECT_NAME} PUBLIC c_std_99 cxx_std_17)
...
   mcap_vendor::mcap
+  pluginlib::pluginlib
+  rcutils::rcutils
+  rosbag2_storage::rosbag2_storage
 )
 
-target_compile_definitions(${PROJECT_NAME} PRIVATE ROS_DISTRO=${ROS_DISTRO})
+if(${rosbag2_storage_VERSION} VERSION_GREATER_EQUAL 0.10.0)
+  target_compile_definitions(${PROJECT_NAME} PRIVATE
+    OVERRIDE_SEEK_METHOD
```
Comment (jtbandes) [inline rosbag2_storage_mcap/CMakeLists.txt]: nit: maybe prefix this e.g. `ROSBAG2_STORAGE_MCAP_OVERRIDE_SEEK_METHOD`?

**Review by jtbandes — APPROVED** [review]

File: mcap_vendor/CMakeLists.txt (hunk 1 above)
Comment (wep21) [reply mcap_vendor/CMakeLists.txt:77]: > Why remove this?

Becaause `ament_export_targets` do the similar thing. https://github.com/ament/ament_cmake/blob/2bf27ce5ad01af340e1f4efd44232c1067d0dbda/ament_cmake_export_targets/cmake/ament_cmake_export_targets_package_hook.cmake#L28-L33

>  Do we need to keep install(TARGETS mcap EXPORT export_mcap)?

Yes, I think we need it.

File: rosbag2_storage_mcap/src/mcap_storage.cpp [hunk 3]
```diff
@@ -63,7 +63,7 @@ class MCAPStorage : public rosbag2_storage::storage_interfaces::ReadWriteInterfa
   /** ReadOnlyInterface **/
   void set_filter(const rosbag2_storage::StorageFilter& storage_filter) override;
   void reset_filter() override;
-#ifdef OVERRIDE_SEEK_METHOD
+#ifdef ROSBAG2_STORAGE_OVERRIDE_SEEK_METHOD
```
Comment (jtbandes) [inline rosbag2_storage_mcap/src/mcap_storage.cpp]:
```suggestion
#ifdef ROSBAG2_STORAGE_MCAP_OVERRIDE_SEEK_METHOD
```

Comment (wep21) [reply rosbag2_storage_mcap/src/mcap_storage.cpp]: https://github.com/ros-tooling/rosbag2_storage_mcap/pull/31/commits/8715b542ccd3208dd061a9a0a24bfb159e3b943d

**Pull request merged**